### DISCUSSION TITLE
CLI: Version command

### DISCUSSION THREAD

**Discussion by CMiksche** [zitadel/zitadel#4329] [Q&A] [answered]
Currently there is no way of showing the current zitadel version. `zitadel version` doesn't exist.

Especially for debugging, that small feature would be helpful.

**Comment by livio-a** [comment]
Hey @CMiksche that makes absolute sense.

Side note: We already display the version in the instance detail
![image](https://user-images.githubusercontent.com/9405495/188798811-ddbdbe31-a1f7-4b0b-991e-f66a06bc6297.png)

And should definitely enable this for the CLI.  I'll convert this into an issue.

**Reply by adlerhurst** [reply]
the zitadel --version flag is implemented in this PR: https://github.com/zitadel/zitadel/pull/4678

**Comment by adlerhurst** [comment, marked as answer]
You can use `zitadel -v` or `zitadel --version` since [v2.12.0](https://github.com/zitadel/zitadel/releases/tag/v2.12.0)
